pkg/orchestrator/orchestrator.go

```diff
@@ -36,13 +36,15 @@ type Orchestrator struct {
 
 func NewOrchestrator(
 	db database.DB,
+	logger log.CtxLogger,
 	pipelines PipelineService,
 	connectors ConnectorService,
 	processors ProcessorService,
 	plugins PluginService,
 ) *Orchestrator {
 	b := base{
 		db:         db,
+		logger:     logger,
```

Comment:
We should set the component in the logger (e.g. `logger.WithComponent("orchestrator")`).